genai-perf/tests/test_console_exporter.py

```diff
@@ -288,49 +296,236 @@ def test_invalid_goodput_output(self, monkeypatch, capsys) -> None:
         returned_data = capsys.readouterr().out
         assert returned_data == expected_content
 
-    @patch(
-        "genai_perf.export_data.console_exporter.ConsoleExporter._construct_table",
-        return_value=None,
-    )
-    @pytest.mark.parametrize(
-        "endpoint_type, metrics, expected_title",
-        [
-            ("chat", LLMMetrics(), "NVIDIA GenAI-Perf | LLM Metrics"),
-            ("embeddings", Metrics(), "NVIDIA GenAI-Perf | Embeddings Metrics"),
-            ("rankings", Metrics(), "NVIDIA GenAI-Perf | Rankings Metrics"),
-            ("vision", LLMMetrics(), "NVIDIA GenAI-Perf | VLM Metrics"),
-            (
-                "image_retrieval",
-                ImageRetrievalMetrics(),
-                "NVIDIA GenAI-Perf | Image Retrieval Metrics",
-            ),
-        ],
-    )
-    def test_console_title(
```

Comment:
Did you intend to delete this test?

Reply:
Did I delete it? Let me check. I didn't wanna delete any test. It might have happened by mistake.